Courses.php

```diff
@@ -106,27 +106,121 @@ function extend_deadline(id) {
+    $title=$_POST["title"];
+    $marks=$_POST["marks"];
+    $Deadline = $deadlinedate." ".$deadlinetime;
+    $date=  date("Y-m-d H:i");
+    
+    $sql = "UPDATE `lab_reports_table` SET `Deadline` = ('" . $Deadline . "'), `Instructions` = ('" . $instructions . "'), `Title` = ('" . $title . "'), `Marks` = ('" . $marks . "') WHERE `lab_reports_table`.`Lab_Report_ID` = '$getid'";
+    if ($con->query($sql) === TRUE) {
+      $_SESSION["info_Updated"]="Information Updated Successfull";
+       
+   } else {
+       echo "Error: " . $sql . "<br>" . $con->error;
+   }
+  }
+  if( $_SESSION['user_type']=="Lecturer"){   
+    $Date=$_SESSION['Date'];
+    $Time=$_SESSION['Time'];
+    $Instructions=$_SESSION['Instructions'];
+    $Title=$_SESSION['Title'];
+    $Marks=$_SESSION['Marks'];
+
+     
+   echo "  <h3><a href='Courses.php?course=".$url."'> Editing Lab Assignment </a></h3>";
+      ?> 
+      <form method='post'   enctype='multipart/form-data' action=''>
+         <input type='hidden' name='frm_uploadlab' value='true' required=''/>
+            <input type='hidden' name='course_id' value='<?php echo ".$id." ?>' required=''/>
+                  <input type='hidden' name='url' value='<?php echo ".$course_url." ?>' required=''/>
+           
+Dealine Date/Time
+<div class='row'> 
+<div class='col-md-7'><input type='date' id='date' name='deadlinedate' placeholder='' class='form-control' required='' value="<?php echo isset($_GET['act']) && $_GET['act']=="edit" ? $Date : ""; ?>"> </div>
+<div class='col-md-5'> <input type='time' class='form-control' name='deadlinetime' value="<?php echo isset($_GET['act']) && $_GET['act']=="edit" ? $Time : ""; ?>"> </div> 
+</div>
+
+Title
+<input type='text'  name='title' placeholder='Ttle' class='form-control' required='' value="<?php echo isset($_GET['act']) && $_GET['act']=="edit" ? $Title : ""; ?>">
+Instructions
+<textarea  name='instructions' placeholder='Assignment Instructions' class='form-control' required='' value="<?php echo isset($_GET['act']) && $_GET['act']=='edit' ? $Instructions : ''; ?>"></textarea>
+Marks
+<input type='text'  name='marks' placeholder='Marks' class='form-control' required='' value="<?php echo isset($_GET['act']) && $_GET['act']=="edit" ? $Marks : ""; ?>">
+Attachment 1
+<input type='file'  name='attachment1' placeholder='Attachment 1' class='form-control'>
+
+Attachment 2
+<input type='file' name='attachment2' placeholder='Attachment 1' class='form-control'>
+
+Attachment 3
+<input type='file'  name='attachment3' placeholder='Attachment 1' class='form-control' >
+
+
+Attachment 4
+<input type='file'  name='attachment4' placeholder='Attachment 4' class='form-control' >
+<br>
+Submission Type  <input type='radio' name='type' value='Individual' required=''> Invidual
+
+<input type='radio' name='type' value='Group' required=''> Group
+<hr>
+<input type='submit' class='btn btn-primary' value='Post Lab Assignment'><br>
+</form><br><br><br><br>
+<?php
+  }}else{
+
```

Comment:
@Teecloudy 
The whole form is reloaded after clicking the **Edit** button?

**Hui**

Reply:
The Posting New report is using a different form with the Edit report. This is because the Posting new form is processed by a script in Script.php and the Edit form is processed by a script in the same file Courses.php
So i couldnt find a better way to make them use one form when they are processed in two different files unless we are going shift the lines from Script.php to Courses.php